vsintegration/Templates.Directory.Build.props

```diff
@@ -31,4 +31,11 @@
     <CopyOutputSymbolsToOutputDirectory>false</CopyOutputSymbolsToOutputDirectory>
   </PropertyGroup>
 
+  <PropertyGroup>
+    <NoOptimizationData>true</NoOptimizationData>
+    <NoInterfaceData>false</NoInterfaceData>
+    <CompressMetadata>false</CompressMetadata>
+    <OtherFlags>$(OtherFlags) --realsig+</OtherFlags>
```

Comment:
Same

Reply:
@vzarytovskii Right at the moment, I am trying out some stuff, none of it is even close to final.